nomad/structs/structs.go

```diff
@@ -9926,10 +9935,6 @@ func (v *Vault) Validate() error {
 	}
 
 	var mErr multierror.Error
-	if len(v.Policies) == 0 {
-		_ = multierror.Append(&mErr, fmt.Errorf("Policy list cannot be empty"))
-	}
```

Comment:
This has to [be checked in the hook now](https://github.com/hashicorp/nomad/pull/18343/files#diff-b0e2575ed29874321dffeb628199cce5e2c9d58e0e1d8842f73d90fcd06386b6R387) because when using identity policies don't need to specified.